ffsync/locker.go

```diff
@@ -8,9 +8,32 @@ import (
 	"github.com/featureform/fferr"
 )
 
-const (
-	UpdateSleepTime = 2 * time.Second
-	ValidTimePeriod = 5 * time.Second
+// TimeWindow is a struct that represents a duration that will be
+// used to sleep as well as valid time period.
+type TimeWindow struct {
```

Comment:
type lockDuration struct

Make it private and give it the same name as duration